deploy/helm/stac-scaler/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: app-stacfastapi
+  namespace: {{ .Release.Namespace }}
```

Comment:
Is there a specific reason why we have the service in potentially a different namespace than everything else? I guess I'm wondering why not just use `{{ .Release.Namespace }}` in all of the other yaml in this chart (and then install into the namespace you want with the `-n` option to helm)? I might have just missed the intent on the namespaces used for this Helm chart, though.

Reply:
Addressed this by using the `.Values.deploymentNamespace` to be consistent with other charts.